packages/components/autocomplete/stories/Autocomplete.stories.tsx

```diff
@@ -172,7 +176,6 @@ export const UsingGroupedItems = (args: AutocompleteProps<GroceryList>) => {
     >
       {/* It’s not necessary to pass "Fruit" (type of one item)  */}
       <Autocomplete<Produce>
-        {...args}
```

Comment:
Any reason why did we remove the `{...args}` here? I think storybook needs that? 🤔

Reply:
was some leftover while playing with the properties